scd/p11token.h

```diff
@@ -0,0 +1,131 @@
+/*
+ * This file is part of GyroidOS
+ * Copyright(c) 2013 - 2024 Fraunhofer AISEC
+ * Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms and conditions of the GNU General Public License,
+ * version 2 (GPL 2), as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GPL 2 license for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, see <http://www.gnu.org/licenses/>
+ *
+ * The full GNU General Public License is included in this distribution in
+ * the file called "COPYING".
+ *
+ * Contact Information:
+ * Fraunhofer AISEC <gyroidos@aisec.fraunhofer.de>
+ */
+
+#ifndef P11TOKEN_H
+#define P11TOKEN_H
+
+#include <stdbool.h>
+#include "pkcs11-lib/pkcs11.h"
```

Comment:
I think this header is not used here, -> move this to p11token.h

Reply:
Correct, header was only needed for struct p11token which is now defined in p11token.c. Will be moved.